docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
+
+It should be noted that this is a demo, and there are dangers to using it in a production environment without further testing.
+Saying that, if you want to do so, and have questions, the team is always available on IRC (`#giantswarm` on `freenode`) <3
+
+## Setup
+- We're using a CoreOS cluster running locally, with Vagrant. See https://coreos.com/os/docs/latest/booting-on-vagrant.html for further information.
```

Comment:
I'd change this then to say sth like `you can use this on any standard fleet cluster, if you want to try it out locally, try the vagrant coreos stuff`